protostar-cli/src/starknet_commands/invoke.rs

```diff
@@ -0,0 +1,64 @@
+use anyhow::{Context, Result};
+use clap::Args;
+
+use starknet::accounts::{Account, Call, SingleOwnerAccount};
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_selector_from_name;
+use starknet::providers::jsonrpc::HttpTransport;
+use starknet::providers::JsonRpcClient;
+use starknet::signers::LocalWallet;
+
+#[derive(Args)]
+#[command(about = "Invoke a contract on starknet")]
+pub struct Invoke {
+    /// Address of contract to invoke
+    #[clap(long)]
+    pub contract_address: String,
```

Comment:
Should we enforce using only long representation? cc @THenry14

Reply:
let's allow for shorts as well